salt/minion.py

```diff
@@ -676,6 +682,7 @@ def eval_master(self,
                     yield tornado.gen.sleep(opts['acceptance_wait_time'])
                 attempts += 1
                 if tries > 0:
+
```

Comment:
Unrelated/accidental edit?

Reply:
Yup.  I think there was a previous attempt here that resulted in that additional newline.